src/domain/Chat/Chat.module.scss

```
@@ -86,55 +113,61 @@
-        color: f.color(orange-700);
-        box-shadow: 0 2px 8px rgba(f.color(orange-700), 0.25);
+        object-fit: cover;
+        display: block;
       }
     }
 
+    // 공통 메시지 스타일
+    %message-bubble {
+      padding: 12px 18px;
+      max-width: 100%;
+      @include m.text-style(md, 500);
```

Stylelint (17.2.0)

[error] Expected empty line before at-rule (at-rule-empty-line-before)

(at-rule-empty-line-before)